src/main/java/com/salesforce/datacloud/jdbc/core/listener/AdaptiveQueryStatusListener.java

```diff
@@ -44,6 +44,7 @@
 import lombok.extern.slf4j.Slf4j;
 import lombok.val;
 
+@Deprecated
```

Comment:
Good opportunity to go from `0.x` to `1.x` by removing these QueryStatusListeners and the entire QueryStatusPoller concept